homeassistant/components/sensor/sense.py

```diff
@@ -53,38 +51,29 @@ def __init__(self, name, sensor_type):
 
 ICON = 'mdi:flash'
 
-MIN_TIME_BETWEEN_DAILY_UPDATES = timedelta(seconds=300)
-MIN_TIME_BETWEEN_ACTIVE_UPDATES = timedelta(seconds=60)
-
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_EMAIL): cv.string,
-    vol.Required(CONF_PASSWORD): cv.string,
     vol.Required(CONF_MONITORED_CONDITIONS):
         vol.All(cv.ensure_list, vol.Length(min=1), [vol.In(VALID_SENSORS)]),
 })
 
+MIN_TIME_BETWEEN_DAILY_UPDATES = timedelta(seconds=300)
+
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Sense sensor."""
-    from sense_energy import Senseable
-
-    username = config.get(CONF_EMAIL)
-    password = config.get(CONF_PASSWORD)
-
-    data = Senseable(username, password)
+    data = hass.data[SENSE_DATA]
 
     @Throttle(MIN_TIME_BETWEEN_DAILY_UPDATES)
     def update_trends():
         """Update the daily power usage."""
         data.update_trend_data()
 
-    @Throttle(MIN_TIME_BETWEEN_ACTIVE_UPDATES)
     def update_active():
         """Update the active power usage."""
         data.get_realtime()
 
     devices = []
-    for sensor in config.get(CONF_MONITORED_CONDITIONS):
+    for sensor in config.get(CONF_MONITORED_CONDITIONS, []):
```

Comment:
CONF_MONITORED_CONDITIONS is a required config parameter so it should always exist, so default to []